docker/px4-dev/scripts/entrypoint.sh

```diff
@@ -23,8 +23,20 @@ else
 	export CCACHE_DISABLE=1
 fi
 
-if [ -f /opt/ros/kinetic/setup.bash ]; then
-	source /opt/ros/kinetic/setup.bash
+# unsetting ROS_DISTRO to silence ROS_DISTRO override warning
+unset ROS_DISTRO
+
+# setup ros1 environment
+if [ -v ROS1_DISTRO ] && [ ! -z "$ROS1_DISTRO" ] && [ -f /opt/ros/$ROS1_DISTRO/setup.bash ]; then
+	source /opt/ros/$ROS1_DISTRO/setup.bash
```

Comment:
We should test this, as I recall sourcing something at entry here won't be available to the running user.

Reply:
https://github.com/osrf/docker_images/blob/master/ros2/bouncy/ubuntu/bionic/ros-core/ros2_entrypoint.sh

Reply:
doesn't this end up being the same as `exec "$@"` as using `exec gosu user "$@"`?

Reply:
I'm not sure, this entire entrypoint idea needs to be reviewed. I need to dig up the details, but it's a problem for current versions of Jenkins and tying us to a specific version of an old plugin (@lamping7). 

The purpose is to enable entering the container as a user with a UID that matches your real host UID. That way you can jump into a container working on the exact same set of files without making a mess of the permissions.

Reply:
I'm not sure if the entrypoint even works as it was for the purpose you said. It would be great to figure this out.

Reply:
I think that's a detail we can address  later, so to not block the PR.